README.md

```diff
@@ -478,6 +501,9 @@ This property previously set the frequency with which Quill polled the DOM for c
 `bounds`
 : Selector or DOM element used by Quill to constrain position of popups. Defaults to `document.body`.
 
+`children`
```

Comment:
You're right, it's not appropriate (however it's harmless).

I'll update the Toolbar example (and fix another formatting issue that I missed as well).